### ISSUE TITLE
Show delay breaks click trigger

### ISSUE THREAD

**Issue by MusicDev33** [drozhzhin-n-e/ng2-tooltip-directive#64]
This works:
```HTML
<span tooltip="Some Value" trigger="click">
```
Now add a show-delay:
```HTML
<span tooltip="Some Value" trigger="click" show-delay="100">
```
The tooltip won't pop up when you add the show delay, it doesn't even get added to the DOM tree. I'm not sure if this is by design, but it's something I ran into.

**Comment by drozhzhin-n-e** [comment]
@MusicDev33 Yes, there is a problem with this. But I would not recommend that you do a delay on click, this will make your interface less responsive.

**Comment by drozhzhin-n-e** [comment]
fixed in version 2.3.12
